src/test/java/gc/cafe/service/ProductServiceImplTest.java

```diff
@@ -0,0 +1,105 @@
+package gc.cafe.service;
+
+import gc.cafe.entity.Products;
+import gc.cafe.util.UUIDConversion;
+import jakarta.persistence.EntityManager;
+import jakarta.transaction.Transactional;
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+
+import java.util.Arrays;
+
+@SpringBootTest
+@Transactional
+class ProductServiceImplTest {
+    @Autowired
+    ProductService service;
+    @Autowired
+    EntityManager entityManager;
```

Comment:
실제 프로덕션 코드와 같은 것을 Autowired하는게 좋아보여요
```suggestion
    @Autowired
    ProductRepository productRepository;
```